www/content/build.md

```diff
@@ -21,6 +21,10 @@ builds:
     # Defaults to the project name.
     id: "my-build"
 
+    # Path to project's (sub)directory containing Go code.
+    # Default is `.`.
+    dir: go
```

Comment:
in the end this will be the build working directory, maybe name it as `cwd` or explain this better on docs? what do you think?

Reply:
Yes, this could be explained better in the docs. I've changed this part to:

```yaml
    # Path to project's (sub)directory containing Go code.
    # This is the working directory for the Go build command(s).
    # Default is `.`.
    dir: go
```
---

I'm slightly more in favor of using `dir` (or `working_dir`) instead of `cwd` since this configuration key would read as _build's directory_ or _build's working directory_ whereas using `cwd` would read as _build's current working directory_ which is a bit awkward in my opinion.